### PR TITLE
home page created

### PR DESCRIPTION
**Pull request by manojncsoni007**

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { useCart } from '../../context/cart-context';
+
+const Navbar = () => {
+    const { cartItem, wishlist } = useCart();
+    return (
+        <nav className="navbar" id="header">
+            <div className="navbar-title">
+                <a href="index.html">
+                    JEWELORY
+                </a>
```
Comment (srishti-maurya) [inline src/components/Navbar/Navbar.jsx:13]: you may use `Link` from react-router-dom, also you may check the spelling `jewellery`

File: src/context/cart-context.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { createContext, useContext, useReducer} from "react";
+
+const CartContext = createContext();
+
+const reducer = (state,action) => {
+    switch(action.type){
+     case 'ADD_TO_CART' : return {...state, cartItem: state.cartItem + 1}
+     case 'ADD_TO_WISHLIST' : return {...state, wishlist: state.wishlist + 1}
```
Comment (srishti-maurya) [inline src/context/cart-context.js:8]: you may add a `default` case in the switch

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    return (
+        <div>
+            <Navbar />
+            <Banner />
+            <main>
+                <h3 className='center'>Category</h3>
+                <div className="homepage-category">
+                    {
+                        category && category.map(({categoryName, img}) => (
+                            <a href="">
```
Comment (srishti-maurya) [inline src/pages/Home/Home.jsx:35]: you may give the `href` attribute a value or instead you may use `Link`

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import Navbar from '../../components/Navbar/Navbar'
+import Banner from '../../components/Banner/Banner'
+import { useCart } from '../../context/cart-context'
+import './Home.css';
+import Footer from '../../components/Footer/Footer'
+
+const Home = () => {
+    const { dispatch } = useCart();
```
Comment (srishti-maurya) [inline src/pages/Home/Home.jsx:11]: you may remove `dispatch` as it isn't used in the file

File: src/pages/Home/Home.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import Navbar from '../../components/Navbar/Navbar'
+import Banner from '../../components/Banner/Banner'
+import { useCart } from '../../context/cart-context'
+import './Home.css';
+import Footer from '../../components/Footer/Footer'
```
Comment (srishti-maurya) [inline src/pages/Home/Home.jsx:8]: you may use barrel exports for components

**Review by srishti-maurya — APPROVED** [review]
Your Deploy failed for the App so you may check the logs and fix the issue accordingly. also, due to which live preview wasn't available, so you might add screenshot for it.

Suggestions:
- You may use `Link` from `react-router` instead if `anchor` tags
- use barrel exports
- store your action types as constants
- give better `alt` attribute values
- you may avoid giving styles directly to the tags

**Review by hrishikeshmane — COMMENTED** [review]
Look ok, other than the issues Srishti pointed out

File: src/pages/Home/Home.jsx (hunk 5 above)
Comment (manojncsoni007) [reply src/pages/Home/Home.jsx:8]: done

File: src/pages/Home/Home.jsx (hunk 4 above)
Comment (manojncsoni007) [reply src/pages/Home/Home.jsx:11]: done

**Pull request merged**